front/app/containers/Authentication/SignUpIn/SignUpInModal/SignIn/PasswordSignin.tsx

```diff
@@ -39,10 +33,11 @@ import tracks from '../tracks';
 
 // style
 import styled from 'styled-components';
-import { viewportWidths } from 'utils/styleUtils';
 
 // typings
 import { ISignUpInMetaData } from '../typings';
+import useFeatureFlag from 'hooks/useFeatureFlag';
```

Comment:
Minor: we could leave a line here such that hooks are not under the `typings` label

Reply:
Personally I prefer to have no sections at all. I don't think it has benefits and only costs time :-)